packages/react-components/react-dialog/etc/react-dialog.api.md

```diff
@@ -91,6 +92,7 @@ export type DialogContextValue = {
     dialogRef: React_2.Ref<DialogSurfaceElement>;
     modalType: DialogModalType;
     requestOpenChange: (data: DialogOpenChangeData) => void;
+    onTransitionStatusChange: (status: DialogTransitionContextValue) => void;
```

Comment:
Please don't expose anything motion related in APIs, it will be migration to new motion APIs soon #30695.

Reply:
Reverted :)